### PR TITLE
Oracle syntax compatibility demo

### PR DESCRIPTION
**Pull request by josh-heyer**
## What Changed?

This combines a static demonstration with a video, walking through two simple queries to illustrate oracle compatibility on EDB Cloud. 

Goal of this demonstration is to serve as an initial introduction to Oracle-compat features on BigAnimal, preparing interested readers for more in-depth topics / discussions.

Supersedes #1864 

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [x] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

**Comment by josh-heyer** [comment]
⚠ Rebased

**Comment by tsteward1** [comment]
I like the video, it's clean and simple and a good example of syntax that's widely used.

File: product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+Type "az" to use Azure CLI
+Type "help" to learn about Cloud Shell
+
+$
+```
+
+The connection string for the demo Advanced Server cluster looks like this:
+
+```
+postgres://demo:password@p-c5kdmd43ehc3kenab0c0.dqrgadbpjqv8pwzl.edbcloud.io:5432/chinook?sslmode=require
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx:27]: I'm pretty sure edbcloud.io should be biganimal.io

**Comment by drothery-edb** [comment]
In terms of placement, I would move it to the end of the Using your cluster section or to the end of the top nav.

File: product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx (hunk 1 above)
Comment (josh-heyer) [reply product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx:27]: Good catch. I... Suspect I'm going to have to rebuild this to correct that though; worth holding off on release you think?

Comment (josh-heyer) [reply product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx]: Updated to biganimal.io (video I'll re-record tonight)

**Pull request merged**